src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/utilities/PointerFinder.java

```diff
@@ -84,43 +84,9 @@ public static PointerLocation find(Address a) {
 
     // Check if address is in the java heap.
     CollectedHeap heap = VM.getVM().getUniverse().heap();
-    if (heap instanceof GenCollectedHeap) {
```

Comment:
Maybe just rename this to `SerialHeap` so that this PR doesn't touch much SA code unnecessarily. (The inconsistency among different collectors can be dealt with in other PRs.)